src/content/resources/glossary.md

```diff
@@ -279,3 +279,33 @@ If not, the pattern _refutes_, or denies, the match.
 Refutable patterns appear in [_matching contexts_][].
 
 [_matching contexts_]: /language/patterns#matching
+
+## Subclass
+
+A _subclass_ is a class that inherits from another class using the
+[`extends`](/language/extend) keyword.
+
+```dart
+class A extends B {} // A is a subclass of B; B is the superclass of A. 
+```
+
```

Comment:
I don't know if it is helpful to mention the following:

Another way to create a subclass is by mixin application: If `M` is a mixin then `class B1 extends A with M {}` has the superclass `A with M` which in turn has the superclass `A`. Similarly, `class B2 = A with M;` is a class that has `A` as its superclass.